gslab_scons/provenance.py

```diff
@@ -101,11 +136,13 @@ def scan(start_path, include_details, include_checksum, detail_limit, file_detai
     return total_size, num_files, last_mtime, file_details, dirs
 
 
-def write_heading(start_path, provenance_path, github_release = None):
+def write_heading(start_path, provenance_path, 
+                  github_release = None,
+                  sig = '*** GSLab directory provenance ***'):
```

Comment:
I'd love it if we defined a sig variable in one location and then passed it to all the functions that use it as a default argument. Hard coding the default in two places is not robust because the two defaults can get out of sync.